includes/wc-order-functions.php

```diff
@@ -93,8 +93,8 @@ function wc_get_orders( $args ) {
  * @return WC_Order|WC_Refund
  */
 function wc_get_order( $the_order = false ) {
-	if ( ! did_action( 'woocommerce_init' ) ) {
-		wc_doing_it_wrong( __FUNCTION__, __( 'wc_get_order should not be called before the woocommerce_init action.', 'woocommerce' ), '2.5' );
+	if ( ! did_action( 'woocommerce_register_post_type' ) ) {
+		wc_doing_it_wrong( __FUNCTION__, __( 'wc_get_order should not be called before post types are registered (woocommerce_register_post_type action).', 'woocommerce' ), '2.5' );
```

Comment:
In my opinion, it would be better to use `woocommerce_after_register_post_type` here.

But since we are not doing that action, we need to do it similarly as it's done for taxonomies [here](https://github.com/woocommerce/woocommerce/blob/master/includes/class-wc-post-types.php#L239).

So [on this line](https://github.com/woocommerce/woocommerce/blob/master/includes/class-wc-post-types.php#L451), we would have:
`do_action( 'woocommerce_after_register_post_type' );`

What do you think?

I'm bringing this up because this still doesn't work and the error message is kind of misleading:
```
<?php
add_action( 'woocommerce_register_post_type', 'test' );
function test() {
    $order = wc_get_order( 90 );
    var_dump( $order );
}
```